import Task

Task.succeed 42

Task.fail "Foo"
